packages/docs/docs/access/binary-security-context.md

```diff
@@ -0,0 +1,16 @@
+# Binary Security Context
+
+When managing access, the FHIR [`Binary`](/docs/api/fhir/resources/binary) resource is unique case. Access controls cannot be applied to [`Binary`](/docs/api/fhir/resources/binary) resources in the same way as other resources, so you must use the `Binary.securityContext` element to add access policies.
+
+The `securityContext` element is a reference to another resource that acts as a proxy for the access controls of that [`Binary`](/docs/api/fhir/resources/binary). For example, if the `securityContext` references a [`Patient`](/docs/api/fhir/resources/patient), then the [`Binary`](/docs/api/fhir/resources/binary) will have that patient's access policy applied to it. Additionally, it will only be viewable by users and resources that have read access to that [`Patient`](/docs/api/fhir/resources/patient).
```

Comment:
> For example, if the `securityContext` references a [`Patient`](/docs/api/fhir/resources/patient), then the [`Binary`](/docs/api/fhir/resources/binary) will have that patient's access policy applied to it. 


This is not correct. Only this part is correct:
> it will only be viewable by users and resources that have read access to that [`Patient`](/docs/api/fhir/resources/patient).